### PR TITLE
[pipeline + external annotators] Minor fixes 

### PR DESCRIPTION
**Pull request by danyaljj**
Two fixes: 
 - I had some view-name inconsistencies in `PathLSTM` --> fixed 
 - Some issues with the model files of Comma-SRL --> fixed.

### CODE REVIEW

File: core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java [hunk 1]
```diff
@@ -69,6 +69,7 @@
     public static final String SRL_NOM = "SRL_NOM";
     public static final String SRL_PREP = "SRL_PREP";
     public static final String SRL_COMMA = "SRL_COMMA";
+    public static final String SRL_VERB_PATH_LSTM = "SRL_VERB_PATH_LSTM";
```
Comment (danyaljj) [inline core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java]: @bhargav here is the name.

Comment (bhargav) [inline core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java]: Thanks. I'll use this when this change is merged and deployed.

**Comment by danyaljj** [comment]
Removed the name from `ViewNames`. Any other comment?

**Comment by bhargav** [comment]
Going forward, we can keep an `ExternalViewNames` files in this project to track all supported views.

**Pull request merged**